core/modules/taxonomy/taxonomy.admin.inc

```diff
@@ -809,7 +809,7 @@ function taxonomy_form_term($form, &$form_state, TaxonomyTerm $term = NULL, Taxo
       $term = entity_create('taxonomy_term', array(
         'vocabulary' => $vocabulary->machine_name,
       ));
-      backdrop_set_title(t('@vocab_name: Add term', array('@vocab_name' => $vocabulary->name)), PASS_THROUGH);
+      backdrop_set_title(t('@vocab_name: Add term', array('@vocab_name' => $vocabulary->name)));
```

Comment:
I think leaving `PASS_THROUGH` is correct, otherwise a vocabulary with quotes or ampersands will get double-escaped when printing the page title.